README.md

```diff
@@ -158,3 +158,91 @@ nessus-agent:
   lookup:
     nessus_groups: 'NessusAgents'
 ```
+
+## Logging Configuration
+
+There is an option to configure a custom log retention policy for each log file that
+is available within the Nessus Agent.  Changing the default log retention policy can
+be done by modifying the `log.json` file.
+
+A custom log retention policy can be configured by setting the following parameters
+for each associated log file:
+
+-   Rotation Strategy
+-   Rotation Time
+-   Max Size
+-   Max Files
+
+The logging configuration is _optional_ and is not required as part of the Nessus Agent
+installation.
+
+For more information, please refer to the vendor documentation:
+
+[How to Manage Nessus log size and rotation](https://community.tenable.com/s/article/How-to-manage-Nessus-log-size-and-rotation)
+
+To configure a custom log retention policy, the parameters must be added to the
+`nessus-agent:lookup` pillar definition in the following structure:
+
+```yaml
+nessus-agent:
+  lookup:
+    log_config:
+      <file-path-to-log-file>:
+        rotation_strategy: ''
+        rotation_size: ''
+        max_size: ''
+        max_files: ''
+```
+
+### `nessus-agent:lookup:log_config`
+
+`log_config` is the main heading that begins the logging configuration.
+
+### `nessus-agent:lookup:log_config:<file-path-to-log-file>`
```

Comment:
I think it's worth noting here that the reporter dict for the specified log file must be added separately to the nessus `log.json` file, or the log config settings will not be added.

Reply:
Okay.  I'll add a note about the additional requirement.